node_modules/oae-principals/lib/api.group.js

```diff
@@ -832,6 +844,119 @@ var updateGroup = module.exports.updateGroup = function(ctx, groupId, profileFie
 };
 
 /**
+ * Delete a group
+ *
+ * @param  {Context}    ctx             Standard context object containing the current user and the current tenant
+ * @param  {String}     groupId         The id of the group to delete
+ * @param  {Function}   callback        Standard callback function
+ * @param  {Obect}      callback.err    An error that occured, if any
+ */
+var deleteGroup = module.exports.deleteGroup = function(ctx, groupId, callback) {
+    var validator = new Validator();
+    validator.check(groupId, {'code': 400, 'msg': 'A valid group id must be provided'}).isGroupId();
+    if (validator.hasErrors()) {
+        return callback(validator.getFirstError());
+    }
+
+    // Check if the user has permission to delete the group
+    canDeleteGroup(ctx, groupId, function(err, canDelete, group) {
+        if (err) {
+            return callback(err);
+        } else if (!canDelete) {
+            return callback({'code': 401, 'msg': 'You are not authorized to delete this group'});
+        }
+
+        // Mark the group as deleted
+        PrincipalsDAO.deletePrincipal(groupId, function(err) {
+            if (err) {
+                return callback(err);
+            }
+
+            // Notify consumers that a group has been deleted
+            PrincipalsEmitter.emit(PrincipalsConstants.events.DELETED_GROUP, ctx, group);
```

Comment:
Can you track this with mixpanel? See `oae-mixpanel/lib/eventlisteners/groups.js`